### PR TITLE
Member Profile Update

### LINKED ISSUE #3057: [Community] Add Badges to Member Profile: Hersh Dillion
**Current Behavior**

@hershd23 has been a consistent contributor to the SMP and Meshery projects.

**Desired Situation**
Let's recognize @hershd23 with the addition of the Service Mesh Performance and Meshery badges on his community member profile at https://layer5.io/community/members/hersh-dhillon

A detailed explanation on how to update a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by bensonOSei**
**Description**

This PR updates the profile of @hershd23 by adding the Mershery and Simple Mersh Performance badges

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.
 

fixes #3057

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit cf5d583b2152ec1c2bcce38cb6df5cf555621820 at: https://62f42acf2f1dff2ab882c68b--layer5.netlify.app

File: src/collections/members/hersh-dhillion/index.mdx [hunk 1]
```diff
@@ -9,4 +9,7 @@ location: New Delhi, India
 bio: I am a CS graduate from IIT Ropar currently working as a Software Engineer in the Platforms and Infrastructure team at Flipkart. My work interests lie in building large scale systems and DevOps. Aside from work you'll generally find me analysing data of football (soccer) teams and fantasy sports leagues or reading philosophy. I love the open culture and people here in the Layer5 community. Hit me up if you wanna chat tech, football, philosophy or all things intersectional.
 status: Active
 published: true
+badges:
+  - meshery
+  - smp
```
Comment (Nikhil-Ladha) [inline src/collections/members/hersh-dhillion/index.mdx:14]: Move it above the `published` frontmater.
`published` should be at last.

**Comment by Nikhil-Ladha** [comment]
And, please sign-pff your commits.
The contributing guide clearly mentions how to do so.

File: src/collections/members/hersh-dhillion/index.mdx (hunk 1 above)
Comment (bensonOSei) [reply src/collections/members/hersh-dhillion/index.mdx:14]: Sure... Will do

**Comment by bensonOSei** [comment]
> And, please sign-pff your commits.
> The contributing guide clearly mentions how to do so.

I've set my text-editor to signoff automatically... It seems there's a problem there. Nonetheless, I'll do that. Thanks😊😊

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**